### PR TITLE
PR: Cleanup and rework: contenttype-icons  ...

### PR DESCRIPTION
**Pull request by fgrcon**
https://github.com/plone/Products.CMFPlone/issues/1226

### CODE REVIEW

File: Products/CMFPlone/tests/testCatalogTool.py [hunk 1]
```diff
@@ -1083,11 +1083,11 @@ def test_provided(self):
         self.assertTrue(IIndexableObjectWrapper.providedBy(w))
         self.assertTrue(IContentish.providedBy(w))
 
-    def test_getIcon(self):
-        doc = self.doc
-        iconname = doc.getIcon(relative_to_portal=1)
-        wrapped = IndexableObjectWrapper(doc, self.portal.portal_catalog)
-        self.assertEqual(wrapped.getIcon, iconname)
+#     def test_getIcon(self):
```
Comment (hvelarde) [inline Products/CMFPlone/tests/testCatalogTool.py]: please remove this code; there's no reason to comment it if is not going to be used.

Comment (fgrcon) [inline Products/CMFPlone/tests/testCatalogTool.py]: removed :-)

**Comment by fgrcon** [comment]
@jensens this PR ist the last one for horrible #1226 ! After merging this we can close the issue!

**Comment by gforcada** [comment]
@fgrcon running tests in http://jenkins.plone.org/job/pull-request-5.0/554/

**Pull request merged**